client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -58,34 +58,35 @@ const HostingFeatures = () => {
 	);
 	const hasEnTranslation = useHasEnTranslation();
 
-	const transfer = useSelector( ( state ) => getAtomicTransfer( state, siteId ) );
-	const isTransferInProgress = [
-		transferStates.PENDING,
-		transferStates.ACTIVE,
-		transferStates.PROVISIONED,
-	].includes( transfer.status );
+	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
+		siteId || undefined
+	);
+	// We have some bug at the backend, when the transfer is complited, but isSiteAtomic is still false, so we need such extra condition
```

Comment:
This comment seems redundant. Will it really help the next dev who comes along, or is this more about communicating the fact that we have yet to resolve https://github.com/Automattic/dotcom-forge/issues/8654?

Reply:
If I were new to the code, I would wonder -- why do we need `siteTransferData?.status === transferStates.COMPLETED;` if we already have `siteTransferData?.isTransferring`... it looks redundant.

That's wy I added the comment, to explain that it's intentional due to some delay at the backend, between updating `isTransferring` and `transferStates.COMPLETED`.

Reply:
```suggestion
	// `siteTransferData?.isTransferring` is not a fully reliable indicator by itself, which is why
	// we also look at `siteTransferData.status`
```
I think we could clarify this comment along these lines